### DISCUSSION TITLE
临时工单区

### DISCUSSION THREAD

**Discussion by zRzRzRzRzRzRzR** [MetaGLM/glm-cookbook#2] [Announcements]
遇到的API问题可以临时在这里提出
反馈问题格式为
```
id: 对话id
question：你遇到的问题
```

**Comment by zRzRzRzRzRzRzR** [comment]
glm api 不具有上网功能

**Comment by Lucas8026** [comment]
id：8469015938265686844
question：大哥，我有一个问题，当我sdk调用glm4模型时，我最开始和它聊天比如就只输入你好，响应挺快的，但是当我把max_tokens最大输出设置为8192时就会非常慢。这是为什么呢？在我理解不应该会这样呀，因为就输入了两个字而已。

另外，对于如果没有设置max_tokens时默认1024最开始很快，但是当上文长度变大，历史记录变长长响应很慢，这个很好理解，但是我不知道问题出自哪里，我是否可以从我这边调用侧代码去优化比如多线程等提高响应速度得到更快的回复（感觉应该是不行），那有什么办法优化吗？

我使用的是流式调用，因为需要实时展示回复内容：
response = client.chat.completions.create(
            model=model,
            messages=conversation,
            stream=True
        )
for chunk in response:
        yield chunk.choices[0].delta.content

麻烦您帮我看一下这两个问题，万分感谢 @ @zRzRzRzRzRzRzR
